<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Soy cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Soy administrador</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>380</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear Reservacion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;MENU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>380</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar Reservacion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Crear Reservacion</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de reservante:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de contacto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de comensales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora Militar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contactoLine">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="comensalesLine">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fechaLine">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="horaLine">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>90</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>250</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="disponibilidad">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Consultar Disponibilidad</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disponibilidad_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de mesa que va a reservar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mesaLine">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modificar Reservacion</string>
     </attribute>
     <widget class="QLineEdit" name="comensalesLine_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="fechaLine_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="horaLine_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreLine_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de contacto:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de comensales:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contactoLine_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modificar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora Militar:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reservacionLine">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de reservante:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Codigo de reservacion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reservacionesList">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ver Reservaciones</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Mesas</string>
     </attribute>
     <widget class="QLineEdit" name="sillasLine">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroMesaLine">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de mesa:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero de sillas:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear Mesa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mesas">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir mesa</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
